utils/appverifier_launch_sample.py

```diff
@@ -0,0 +1,149 @@
+    subprocess.run(args=appverif_xml_dump_args)
+
+    xml_result = appverifier_xml.parseXML(tmp_xml_file_path, True)
+    if (xml_result != 0):
+        print (f"ERROR: XML parse returned failure!")
+    return xml_result
+
+
+def booleanString(string):
+    string = string.lower()
+    if string not in {"false", "true"}:
+        raise ValueError("Boolean is not true or false!")
+    return string == "true"
+
+
+def main():
+    argument_parser = argparse.ArgumentParser(
+        description="AppVerifier XML output util")
+    argument_parser.add_argument("--sample_file", metavar="<C:\\example\\sample.exe>",
+                                 required=True, default="", help="Sample to launch that AppVerifier is following")
+    argument_parser.add_argument("--sample_region", metavar="<Name of region>",
+                                 required=False, default="us-east-1", help="The name of the region to use for accessing secrets")
+    argument_parser.add_argument("--sample_secret_endpoint", metavar="<Name of endpoint secret>",
+                                 required=False, default="unit-test/endpoint", help="The name of the secret containing the endpoint")
+    argument_parser.add_argument("--sample_secret_certificate", metavar="<Name of certificate secret>", required=False,
+                                 default="unit-test/certificate", help="The name of the secret containing the certificate PEM file")
+    argument_parser.add_argument("--sample_secret_private_key", metavar="<Name of private key secret>", required=False,
+                                 default="unit-test/privatekey", help="The name of the secret containing the private key PEM file")
+    argument_parser.add_argument("--sample_arguments", metavar="<Arguments here in single string!>",
+                                 required=False, default="", help="Arguments to pass to sample")
+
+    parsed_commands = argument_parser.parse_args()
+
+    print ("Registering with AppVerifier")
+    app_verifier_tests = ["Exceptions", "Handles", "Heaps", "Leak", "Locks", "Memory", "SRWLock", "Threadpool", "TLS"]
+    registerAppVerifier(parsed_commands.sample_file, app_verifier_tests)
+
+    print("Starting to launch sample...")
+    sample_result = launchSample(
+        parsed_commands.sample_file,
+        parsed_commands.sample_region,
+        parsed_commands.sample_secret_endpoint,
+        parsed_commands.sample_secret_certificate,
+        parsed_commands.sample_secret_private_key,
+        parsed_commands.sample_arguments)
+    
+    print ("Parsing XML...")
+    tmp_xml_file_path = os.path.join(tempfile.gettempdir(), "tmp.xml")
+    xml_result = checkAppVerifierXML(parsed_commands.sample_file, tmp_xml_file_path)
+
+    print ("Unregistering with AppVerifier")
+    unregisterAppVerifier(parsed_commands.sample_file)
+
+    if (xml_result != 0):
+        sys.exit(xml_result)
+    sys.exit(sample_result)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
trivial: Github is pointing out that this file doesn't end with a blank line

If you use VSCode, turn on this option in your global User Settings:

Files: Insert Final Newline - ("files.insertFinalNewline" in json view)